api/bus.go

```diff
@@ -206,17 +206,18 @@ type Object struct {
 
 // ObjectMetadata contains various metadata about an object.
 type ObjectMetadata struct {
-	Name   string  `json:"name"`
-	Size   int64   `json:"size"`
-	Health float64 `json:"health"`
+	Name     string  `json:"name"`
+	Size     int64   `json:"size"`
+	Health   float64 `json:"health"`
+	MimeType string  `json:"mimeType"`
 }
 
 // ObjectAddRequest is the request type for the /object/*key endpoint.
 type ObjectAddRequest struct {
-	Bucket        string                                   `json:"bucket"`
-	ContractSet   string                                   `json:"contractSet"`
-	Object        object.Object                            `json:"object"`
-	UsedContracts map[types.PublicKey]types.FileContractID `json:"usedContracts"`
+	Bucket string        `json:"bucket"`
+	Object object.Object `json:"object"`
+
+	object.ObjectMetadata
```

Comment:
This seems a bit odd to me. Feels like the `ObjectMetadata` would be part of `Object`. Especially since the metadata has a `Name` field and the object has a `Key` field which are essentially the same. Making using this endpoint a bit strange. 

Also feels a bit like `api.ObjectMetadata` and `object.ObjectMetadata` are same same but different.

Maybe we can just use `api.Object` in the `ObjectAddRequest` and extend `api.ObjectMetadata`?